src/mixins/_mixins.scss

```diff
@@ -308,6 +308,10 @@
     color: var(--sapLink_InvertedColor);
   }
 
+  &--subtle {
```

Comment:
Shouldn't this `link` mixin be extracted to separate link component file?

Reply:
@salarenko we have an inverted one here. How do you see the subtle to be in another file?